index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" type="text/css" href="styles.css" media="screen"/>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/9650a62e47.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="css/style.css">
+  <title>Product Landing Page - iMouse</title>
+</head>
+

```

Comment:
Kindly remove the extra blank lines and indentations for a cleaner code file.
Please keep in mind that we're only adding blank lines to separate two logical pieces of code or two large sections apart.